test/test.tools.js

```diff
@@ -45,3 +47,40 @@ describe("Monkeypatching", () => {
         assert.equal("nested patched orig", result);
     });
 });
+
+describe("Test tools for parsing new gmail body_params", () => {
+    const gmail = new Gmail();
+    const data =  JSON.parse(testData.new_gmail_archive_action_body_params);
+    const mockMessageIds = ['msg-f:1600724307680265309', 'msg-f:1600724938213937205', 'msg-f:1600725174437456906', 'msg-f:1600725319255992336', 'msg-f:1600725448529658711'];
```

Comment:
This global test-variable is only used inside one test. Putting it inside that test would IMO make the test easier to read.